.eslintignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+client/dist
```

Comment:
```suggestion
dist
```

Let’s ignore any `dist` dir since later we will have `react-client` and `vue-client`